src/App.js

```diff
@@ -2,12 +2,17 @@ import "./App.css";
 // import MockAPI from './mockman/MockAPI'
 import { Routes, Route } from "react-router-dom";
 import "./components/navbar.css"
+import './components/login/login.css'
+import './components/SignIn/signIn.css'
 import LandingPage from "./page/landingPage";
 import SingleVideoPage from "./page/SingleVideoPage";
 import VideoListing from "./page/videoListing";
 import WatchLaterpage from "./page/WatchLaterpage";
 import  HistoryPage  from "./page/History-page";
 import LikeVideoPage from "./page/LikeVideoPage";
+import SignIn from "./components/SignIn/SignIn";
+import Login from "./components/login/Login";
+import PlayList from "./components/PlayList";
```

Comment:
You can export all components and pages into their respective index.js file and then import from  that file.